1-mandatory/3-slideshow/index.html

```diff
@@ -12,6 +12,19 @@
   </head>
   <body>
     <!-- Write your HTML in here -->
+    <div id="container">
+      <img id="image" src="assets/images/pic0.jpg" id="currentImage" alt="" />
+      <button onclick="autoBackward()">Auto Back</button>
```

Comment:
It's really uncommon to see the event binding as part of the HTML, mostly to separate out the display logic from the behavioural/interaction logic. You should probably have used `addEventListener` function calls instead.

Reply:
Totally agree, I think this was just because of the laziness